reference/catalog/category-trees_catalog.v3.yml

```diff
@@ -926,8 +936,102 @@ components:
       minLength: 1
       type: string
       description: |-
-        The name displayed for the category tree. The name must be unique with respect to the category tree of siblings on the same level (you can duplicate the name for other siblings on another level). Required in a POST.
+        The name displayed for the category. Name is unique with respect to the categoryʼs siblings.
+        Required in a POST.
       example: Bath
+    NotFoundError:
+      required:
+        - status
+        - title
+        - type
+      title: Not Found
+      type: object
+      properties:
+        status:
+          type: integer
+          description: The HTTP status code.
+        title:
+          type: string
+          description: The error title describing the particular error.
+        type:
+          type: string
+    GeneralError:
+      required:
+        - status
+        - title
+        - type
+      title: ''
+      type: object
+      properties:
+        status:
+          type: integer
+          description: The HTTP status code.
+        title:
+          type: string
+          description: The error title describes the particular error.
+        type:
+          type: string
+        code:
+          type: integer
+          description: The custom code of the error.
+    GeneralErrorWithErrors:
+      required:
+        - status
+        - title
+        - type
+        - errors
+      title: ''
+      type: object
+      properties:
+        status:
+          type: integer
+          description: The HTTP status code.
+        title:
+          type: string
+          description: The error title describing the particular error.
+        type:
+          type: string
+        errors:
+          title: Detailed Errors
+          type: object
+          properties: { }
+          additionalProperties: true
+  responses:
+    NotFoundError:
```

Comment:
Why is this schema is needed? It is only listed under schemas and responses but is not called in any of the endpoints.

Reply:
Removed